content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -263,7 +264,7 @@ def cloud_storage(args_hash)
               flavor.root_disk_size.to_i + flavor.ephemeral_disk_size.to_i + flavor.swap_disk_size.to_i
             end
 
-  storage += cloud_volume_storage(args_hash)
+  storage += cloud_volume_storage(args_hash) if @reconfigure_request
```

Comment:
@billfitzgerald0120 
Is this related to this PR? or related to configure request

Reply:
@mkanoor I added the "if @reconfigure_request" because it caused the test to fail without it.